test/test_browser.py

```diff
@@ -5408,6 +5408,14 @@ def test_audio_worklet_post_function(self, args):
   def test_audio_worklet_modularize(self, args):
     self.btest_exit('webaudio/audioworklet.c', args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-sMODULARIZE=1', '-sEXPORT_NAME=MyModule', '--shell-file', test_file('shell_that_launches_modularize.html')] + args)
 
+  # We can't test playback, so don't need the data files, we're testing it builds and runs
+  @parameterized({
+    '': ([],),
+    'minimal_with_closure': (['-sMINIMAL_RUNTIME', '--closure', '1', '-Oz'],),
```

Comment:
We normally write this `--closure=1` as a single arg

Reply:
I just copied an existing test for this, I'll change it tomorrow.